.github/workflows/ui-e2e.yaml

```
@@ -0,0 +1,520 @@
+          tag: ${{ inputs.tag }}
+          memory-limit: ${{ env.KIND_MEMORY_LIMIT }}
+          quay-user: ${{ secrets.QUAY_RHACS_ENG_RO_USERNAME }}
+          quay-pass: ${{ secrets.QUAY_RHACS_ENG_RO_PASSWORD }}
+
+      - name: Deploy StackRox (background)
+        uses: ./.github/actions/deploy-stackrox
+        with:
+          tag: ${{ inputs.tag }}
+          scanner: v2
+          github-token: ${{ secrets.RHACS_BOT_GITHUB_TOKEN }}
+          quay-user: ${{ secrets.QUAY_RHACS_ENG_RO_USERNAME }}
```

actionlint (1.7.12)

[error] input "quay-user" is not defined in action "Deploy StackRox" defined at "./.github/actions/deploy-stackrox". available inputs are "background", "cluster-name", "github-token", "registry-password", "registry-username", "resources", "roxie-version", "scanner", "tag"

(action)